src/views/collection-view/components/collection-meta/collection-meta.module.css

```diff
@@ -0,0 +1,45 @@
+.icon {
+  color: var(--token-color-foreground-primary);
+}
+
+.heading {
+  margin-top: 12px;
+}
+
+.description {
+  margin-top: 8px;
+  color: var(--token-color-foreground-primary);
+}
+
+.cta {
+  display: flex;
+  align-items: center;
+  margin-top: 24px;
+  margin-bottom: 16px;
+
+  @media (--dev-dot-tablet-up) {
+    margin-top: 32px;
+    margin-bottom: 24px;
+  }
+}
+
+.ctaButton {
+  composes: root from '/src/components/button/button.module.css';
+  composes: primary from '/src/components/button/button.module.css';
+  composes: medium from '/src/components/button/button.module.css';
+  & a {
```

Comment:
Possibly blocking: I think the `a` here looks odd when focused:

<img width="101" alt="CleanShot 2022-04-20 at 11 20 59@2x" src="https://user-images.githubusercontent.com/4624598/164265543-637f9f3b-f4ee-4488-a163-23351578c205.png">

☝️ This is in Firefox, so you can see the focus ring, but in Safari I find it _really_ hard to see (not even sure if it's there? but am sure I'm focusing it, since `Enter` navigates as expected):

<img width="98" alt="CleanShot 2022-04-20 at 11 22 38@2x" src="https://user-images.githubusercontent.com/4624598/164265820-96d5939f-782e-4d84-b970-848de6ebb01b.png">

Maybe a potential fix: `compose`-ing the above classNames directly onto the `a` element might work? (Though I suspect that might break in some other way, so maybe a terrible suggestion, I'm not sure 😬 ). Possibly related to the above, very speculative suggestion of setting up a `components/button-link` component, & copy-pasting styles into there to make a more isolated component focused on the "`<a />` that looks like a `button`" use case.